layouts/index.html

```diff
@@ -15,7 +15,11 @@
         {{ partial "sections/ticket.html" (dict "tickets" .Site.Data.tickets.data) }}
     {{ end }}
 
-
+    <!-- Schedule -->
+    {{ if .Site.Data.schedule.live }}
+    {{ partial "sections/schedule-short.html" }}
```

Comment:
there is no file named 'sections/schedule-short.html'. we wanted to use this https://github.com/pythonindia/inpycon2021/blob/main/layouts/partials/sections/schedule.html I think?

Reply:
There is a file named `sections/schedule-short.html` that is added as part of this PR. https://github.com/pythonindia/inpycon2021/pull/55/files#diff-7a0ac87803f94fcca38390e72cedb6390e44d092565f13a7fd01ef9ed0939d8a

Reply:
@raukadah My PR is adding that partial: `sections/schedule-short.html`. Check this out: layouts/partials/sections/schedule-short.html in my branch

The original partial can't be used because changes were needed to be made to that partial. And, if I did make the changes in the original one then the separate schedule page would be get mangled as well. Hence, the need to clone part of the schedule.html partial (original) and create a short version of it.

Reply:
Ah, sorry my bad, I missed that file. Thank you @SaptakS for pointing it out. Thank you for putting it together @vipulgupta2048 :-)

Reply:
Resolving